.ci/scripts/run_tests_ucc_mpi.sh

```diff
@@ -101,6 +108,18 @@ for MT in "" "-T"; do
     mpirun $(mpi_params $PPN 1) $ucx_tls_no_cuda_ipc $tlcuda_args $EXE $MT $TG --mtypes cuda -c $tlcuda_colls
     echo "INFO: UCC MPI unit tests (TL/CUDA) ... DONE"
 
+    echo "INFO: UCC MPI unit tests (TL/MLX5) ..."
+    # shellcheck disable=SC2086
+    if [ "x$CX7_DEV" == "x" ]; then
+        echo "No active CX7 devices found on $HEAD_NODE"
+    elif [ $NNODES -lt 2 ]; then
+        echo "At least two nodes are required, but only $NNODES are available"
+    else
+        tlmlx5_args=" -x UCC_CLS=basic -x UCC_CL_BASIC_TLS=ucp,mlx5 -x UCC_TL_MLXS_NET_DEVICES=$CX7_DEV:1 -x UCC_TL_MLX5_TUNE=inf -x UCX_RC_MLX5_DM_COUNT=0 -x UCX_DC_MLX5_DM_COUNT=0 "
```

Comment:
why set DM_COUNT?

Reply:
Ok, up to you. I just think having default config testing has bigger priority

Reply:
Ok that makes sense. So let me remove it for now and once a2a is back up and running maybe we'll simply test with and without those options.